r/R/expression.R

```diff
@@ -43,7 +43,17 @@
   "yday" = "day_of_year",
   "hour" = "hour",
   # second is defined in dplyr-functions.R
-  "minute" = "minute"
+  "minute" = "minute",
+  "log" = "ln_checked",
```

Comment:
Now that these exist, we should define the `Math` group generic for Arrow objects, like we have for `Ops`. I made https://issues.apache.org/jira/browse/ARROW-13337 for that.

Reply:
Hmm, in R, `log` is not unary: 

```
log <- function (x, base = exp(1))
```

So we should handle that correctly. If arrow doesn't support general `log_baseN`, (1) make sure there's a JIRA for that, and (2) map the cases that do exist (e, 10, 2, etc.) and for now error "not supported" for any other value of base.

Reply:
Done and opened https://issues.apache.org/jira/browse/ARROW-13345